### DISCUSSION TITLE
Aprendendo usar o Blazor c/ Mudblazor

### DISCUSSION THREAD

**Discussion by gfmolon** [balta-io/forum#650] [Tira Dúvidas] [answered]
Oi, tentei configurar o mudblazor e não consegui carregar corretamente o css etc, poderiam dar uma olhada?

Esqueci de algum detalhe? Tentei usar card, container etc e o css não vem.

https://github.com/gfmolon/DemoMud

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @gfmolon, como vai?

Claro, posta os detalhes aí com print dos erros, com os trechos de código que usou para configurar o MudBlazor e detalhando melhor o que não está configurado corretamente por favor!

**Reply by gfmolon** [reply]
![Screenshot 2024-08-08 at 13 29 20](https://github.com/user-attachments/assets/1736e6a0-6203-4e17-b750-b21024a48fa6)

MainLayout está assim:
`@inherits LayoutComponentBase

<MudThemeProvider />
<MudPopoverProvider />
<MudDialogProvider />
<MudSnackbarProvider />

<MudLayout>
    <MudMainContent>
        <MudContainer>
            <div class="d-flex justify-center pb-8 pt-8">
                <MyNavMenu />
            </div>
            <div class="d-flex justify-center pb-8 pt-8">
                <div>@Body</div>
            </div>
        </MudContainer>
    </MudMainContent>
</MudLayout>
`

App.razor
`    <link href="https://font`enter code here`s.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
        rel="stylesheet" />
    <link href="_content/MudBlazor/MudBlazor.min.css" rel="stylesheet" />`

Todo o código está em https://github.com/gfmolon/DemoMud

**Comment by gfmolon** [comment]
Não deveria aparecer o css de container igual ta no site do mudblazor? Obrigado. Esqueci alguma coisa?

**Reply by andrebaltieri** [reply]
@gfmolon tem um problema no seu CSS:
```
<link href="https://font`enter code here`s.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
        rel="stylesheet" />
```

Saiu um `enter code here` na URL 😅

**Reply by gfmolon** [reply]
Blz!! kk.

**Reply by gfmolon** [reply]
Ainda sim nao carregou o css correto, mesmo usando 

<MudLayout>
    <MudMainContent>
        <MudContainer>

**Reply by gfmolon** [reply]
coloquei @using DemoMud.Components
@using MudBlazor nos arquivos de layout parou dar erro no visual studio, mas mesmo assim nao carregou estilo. Desculpa a duvida deve ser algo simples mas enfim.. abs!!

**Reply by andrebaltieri** [reply]
@gfmolon bom dia, como vai?

Comparei seu source aqui e seu LINK para font ainda está errado!

Acredito que o Mud está funcionando, só a font esteja errada... assim que corrigir vai funcionar!

**Reply by gfmolon** [reply]
Funcionou! só a cor do fundo parece que não trouxe o azul.

**Comment by gfmolon** [comment]
![Screenshot 2024-08-12 at 13 25 37](https://github.com/user-attachments/assets/0ca74bee-8525-4ad4-9c1c-72beb5e74763)

Realmente o design é muito bonito, um simples menu fica bem legal né. Só falta o detalhe do fundo agora.

**Reply by andrebaltieri** [reply]
Wowwwwww... meus parabéns!!!! Já já sai um produto aí 🚀
